format_spec/fragment.md

```diff
@@ -137,7 +138,8 @@ The tile offsets is a [generic tile](./generic_tile.md) with the following inter
 
 ### Tile Sizes
 
-The tile sizes is a [generic tile](./generic_tile.md) with the following internal format:
+The tile size refers to the _on-disk_ size. \
```

Comment:
Those actually refer to the in memory tile sizes. This information needs to be stored because there is no way to calculate how big a var tile is in memory.